### ISSUE TITLE
Support React 16.0.0-alpha.6 

### ISSUE THREAD

**Issue by CodeXtinction** [react-native-community/react-native-svg#305]
Hello, im trying to install react-native-svg in a react-native 0.43.3 project and i got this warn. And yes React 16.0.0-alpha.6  is installed.

```
react-native-svg@5.1.7 requires a peer of react@>=15.4.0 but none was installed.

```

**Comment by geecurve** [comment]
Same build versions, same problem.

**Comment by williamwlchan** [comment]
Same problem here. If I run `npm install --save react@15.4.0`, then it pops me up a message "requires a peer of react@>=16.0.0-alpha.6 but none was installed." This is like a loophole.

**Comment by abb1991** [comment]
I have the same problem.

**Comment by anthonied** [comment]
Same here

**Comment by magicismight** [comment]
Fix in 5.1.8

**Comment by aidan2129** [comment]
@magicismight Not for me still getting the same error

```
react-native-svg@5.1.8 requires a peer of react-native@>=0.44.0 but none was installed.
npm WARN react-native-svg@5.1.8 requires a peer of react@16.0.0-alpha.6 but none was installed
```
